website/source/docs/commands/environment-variables.html.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: "docs"
+page_title: "Environment Variables"
+sidebar_current: "docs-commands-environment-variables"
+description: |-
+  Consul's behavior can be modified by certain environment variables.
```

Comment:
We still need a slightly longer description here. Copying the first paragraph is usually a good idea.